azurerm/internal/services/datafactory/data_factory_resource.go

```diff
@@ -70,12 +73,24 @@ func resourceDataFactory() *schema.Resource {
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
 								"SystemAssigned",
+								"UserAssigned",
 							}, false),
 						},
+
+						"user_identity_ids": {
```

Comment:
could we make this 
```suggestion
						"identity_ids": {
```

to be consistent with the rest of the provider?

Reply:
done :)